Src/Microsoft.Dynamic/Microsoft.Dynamic.csproj

```diff
@@ -1,39 +1,46 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <Import Project="$(ProjectDir)..\..\Build\Common.proj" />
+
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(TargetFrameworks)' == ''">net40;net45;netcoreapp2.0</TargetFrameworks>
-    <PlatformTarget>anycpu</PlatformTarget>
-    <ProjectGuid>{EB66B766-6354-4208-A3D4-AACBDCB5C3B3}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFrameworks>net40;net45;netcoreapp2.0</TargetFrameworks>
     <RootNamespace>Microsoft.Scripting</RootNamespace>
```

Comment:
Not an issue with this PR, but I don't think should be Microsoft.Scripting. We can probably get rid of it.

Reply:
All the files in this project use `Microsoft.Scripting` as the root namespace. Removing this property means that when creating new files in this project the namespace will be based on `Microsoft.Dynamic` instead.

Reply:
You're right, I hadn't checked the source files. We can leave it as is.